cmd/kpod/inspect.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"github.com/kubernetes-incubator/cri-o/cmd/kpod/formats"
 	"github.com/kubernetes-incubator/cri-o/libkpod"
-	libkpodimage "github.com/kubernetes-incubator/cri-o/libkpod/image"
+	"github.com/kubernetes-incubator/cri-o/libpod/images"
```

Comment:
can we add libpod here like we do in images.go to stay consistent?  Or remove it from there?

Reply:
I am working on a full rewrite to eliminate the use of these.  Basically use Runtime instead of using lower level functions.